### PR TITLE
Suite writing tutorial: better syntax, separate Fortran

### PR DESCRIPTION
**Pull request by benfitzpatrick**
The syntax is better and I've moved the Fortran into an appendix and subdirectory. I've also made a couple of corrections and minor improvements.

@oliver-sanders, please review. Comments welcome from @hjoliver as well!

### CODE REVIEW

**Comment by hjoliver** [comment]
Looks good Ben.  I might make some minor comments later tonight if I have the energy, otherwise it can wait for your next iteration.

**Comment by benfitzpatrick** [comment]
Now added shading back in. @oliver-sanders, I have the right package file in my `~/cylc-tutorial/` directory which you'll need to build this.

File: intro-section.tex [hunk 1]
```diff
@@ -75,9 +75,12 @@ \subsection{Hello World in cylc}
 \terminology{A cylc \underline{suite daemon} is a light-weight server program
 dedicated to managing a single workflow.}
 
-\paragraph*{Demo}
-Copy the Hello World example code into a suite.rc file in a directory called
-hello\_world, and run the following commands:
+\begin{shaded*}
+
+\subsection{Hello World Tutorial}
```
Comment (oliver-sanders) [inline intro-section.tex:78]: The shaded environment doesn't seem to play nicely with headings (it shades the vertical space before the heading).

Visually this works ok:

``` latex
\vspace{5mm}
\begin{shaded*}
{\textbf{\large{Hello World Tutotial}}}
```

... but obviously doesn't appear in the document structure.

File: intro-section.tex [hunk 2]
```diff
@@ -75,9 +75,12 @@ \subsection{Hello World in cylc}
...
+
+Copy the Hello World example code above into a {\em suite.rc} file in a directory called
+{\em hello\_world}, and run the following commands:
 
 \begin{lstlisting}[language=bash]
```
Comment (oliver-sanders) [inline intro-section.tex:85]: The default code colour is quite similar to the shaded background making it a little difficult to read listings.

Comment (oliver-sanders) [reply intro-section.tex:85]: Addressed by    41d4a78

File: intro-section.tex (hunk 1 above)
Comment (oliver-sanders) [reply intro-section.tex:78]: Addressed by    41d4a78

**Comment by oliver-sanders** [comment]
Looks good.

**Pull request merged**